livereload.js

```diff
@@ -4,21 +4,19 @@ const { normalize } = require("path");
 
 // 'disconnect' is not needed as it will only be raised, when either the child process or this process calss disconnect explicitly.
 // We are abel to ensure, that this will never happen, as we do not need communication with the spawned subprocess
-const terminationSignal = ["close", "exit"];
+const terminationSignals = ["close", "exit"];
 
 function startSubprocess() {
   return new Promise((res, rej) => {
     const [command, ...args] = process.argv.slice(2);
     const proc = spawn(normalize(command), args, {

```

Comment:
There's no ideal way to approach this without building a full parser for `qode`'s args (technically we could take this straight out of the node library itself, as it's the same CLI interface), but at the very least, we'll need to normalise the first argument, `./dist/index.js` of the `args` array, as it also contains a path.

Reply:
I checked it on my machine and it works as it is now. But if you feel more comfortable normalizing all args, we can change it.

Reply:
Oh, interesting. Maybe `qode`/`node` normalises the args under-the-hood anyway, so for this particular executable we don't have to normalise them before passing them. Brilliant. If it works with macOS paths on your Windows machine without being normalised by the livereload script, then I have no further concerns!

Reply:
Yeah, it seems like it resolves under the hood. I checked other projects and I always read files using "/" as path seperator without any problems,